build/docker/dockerfile.tmpl

```diff
@@ -0,0 +1,71 @@
+
+#----------------------------------------------------------------------------------------------
+FROM redisfab/redis:{{REDIS_VERSION}}-{{ARCH}}-{{OSNICK}} AS redis
+FROM {{OS}} AS builder
+
+SHELL ["/bin/bash", "-l", "-c"]
+
+RUN echo "Building for {{OSNICK}} ({{OS}}) for {{ARCH}} [with Redis {{REDIS_VERSION}}]"
+
+WORKDIR /build
+COPY --from=redis /usr/local/ /usr/local/
+
+ADD . /build
+
+RUN ./deps/readies/bin/getupdates
+RUN ./deps/readies/bin/getpy2
+RUN ./sbin/system-setup.py
+
+RUN /usr/local/bin/redis-server --version
+RUN make fetch
+
+RUN make build SHOW=1
+
+{% if PACK == "1" %}
+RUN make build LITE=1 SHOW=1
+RUN make build COORD=rlec SHOW=1
+{% endif %}
+
+{% if TEST == "1" %}
+RUN make build COORD=oss SHOW=1
```

Comment:
It's awkware that TEST specifies the COORD. Maybe that should be separate? WDYT?

Reply:
The OSS coordinator is sort of missing in action in the Docker world (there is no OSS coordinator in Docker at the moment). However, if we request the Docker build to include tests, we run the full set of tests, which includes the OSS coordinator.